### ISSUE TITLE
Cannot read property 'theme' of undefined

### ISSUE THREAD

**Issue by aaronjudd** [reactioncommerce/reaction#3080]
Found when processing order with Shippo + Stripe. RC -v 1.5.0-rc 4

```
Exception in setTimeout callback: TypeError: Cannot read property 'theme' of undefined
    at addBodyClasses (http://localhost:3000/app/app.js?hash=12a5ed472ddeed8375cee0e6417dc417a3f45e83:61459:38)
    at http://localhost:3000/app/app.js?hash=12a5ed472ddeed8375cee0e6417dc417a3f45e83:102848:18
    at Array.forEach (<anonymous>)
    at Object.run (http://localhost:3000/app/app.js?hash=12a5ed472ddeed8375cee0e6417dc417a3f45e83:102847:26)
    at BrowserRouter._this.handleLocationChange (http://localhost:3000/app/app.js?hash=12a5ed472ddeed8375cee0e6417dc417a3f45e83:104342:23)
    at listener (http://localhost:3000/packages/modules.js?hash=17e4db972c5830b17a42c3595b848cbf72ec3e0e:89163:24)
    at http://localhost:3000/packages/modules.js?hash=17e4db972c5830b17a42c3595b848cbf72ec3e0e:89182:23
    at Array.forEach (<anonymous>)
    at Object.notifyListeners (http://localhost:3000/packages/modules.js?hash=17e4db972c5830b17a42c3595b848cbf72ec3e0e:89181:15)
    at setState (http://localhost:3000/packages/modules.js?hash=17e4db972c5830b17a42c3595b848cbf72ec3e0e:88728:23)```

**Comment by spencern** [comment]
I just ran through an order and processed it and was unable to reproduce this, but have found the code that was causing the problem and pushed up a fix.

**Comment by spencern** [comment]
Resolved by #3086
